electrumsv/wallet.py

```diff
@@ -988,7 +1062,10 @@ def get_address_history(self, address: Address):
                     self.apply_transactions_xputs(tx_id, tx)
 
         self.txs_changed_event.set()
-        await self._trigger_synchronization()
+        # --> awakens network._monitor_txs loop
+        # Any transactions that do not 'HasByteData' --> "missing_transactions" pool
+        # to get ByteData and only then do they upgrade --> StateCleared
+        await self._trigger_synchronization()  # ? rename to 'maintain_addresses_gap'?)
```

Comment:
No, this is correct. Synchronisation is the process of fetching missing data, in the form of both tx bytedata and proofs. The addresses gap is a concept that is going away.